tarogarden/planter.go

```diff
@@ -554,3 +555,9 @@ func (c *ChainPlanter) CancelSeedling() error {
 // A compile-time assertion to make sure that ChainPlanter implements the
 // taronursery.Planter interface.
 var _ Planter = (*ChainPlanter)(nil)
+
+// ErrDuplicateSeedlingName is an error type for use with asset seedling name
+// unique constraint violations.
+var ErrDuplicateSeedlingName = errors.New(
```

Comment:
nit: This should be defined at the top of the file instead. Or maybe even in `interface.go`?

Reply:
Moved to the top of `interface.go`. Let me know if that's not quite the right place. In general, do you prefer to define errors at the top of a file?